docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md

```diff
@@ -119,12 +117,56 @@ During a high-availability backup, the entire YugabyteDB Anywhere state is copie
 
 All instances involved in high availability should be of the same YugabyteDB Anywhere version. If the versions are different, an attempt to promote a standby instance using a YugabyteDB Anywhere backup from an active instance may result in errors.
 
-Even though you can perform an upgrade of all YugabyteDB Anywhere instances simultaneously and there are no explicit ordering requirements regarding upgrades of active and standby instances, it is recommended to follow these guidelines: 
+Even though you can perform an upgrade of all YugabyteDB Anywhere instances simultaneously and there are no explicit ordering requirements regarding upgrades of active and standby instances, it is recommended to follow these general guidelines: 
 
 - Start an upgrade with an active instance.
 - After the active instance has been upgraded, ensure that YugabyteDB Anywhere is reachable by logging in and checking various pages.
 - Proceed with upgrading standby instances. 
 
+The following is the detailed upgrade procedure:
+
+1. Stop the high-availability synchronization. This ensures that only backups of the original YugabyteDB Anywhere version are synchronized to the standby instance.
+
+2. Upgrade the active instance. 
+
+   Expect a momentary lapse in availability for the duration of the upgrade.
+
+3. If upgrade completes, proceed to step 4. Otherwise, proceed to step 9.
+
+4. On the upgraded instance, perform post-upgrade validation tests that may include creating or editing a universe, backups, and so on. If validation is successful, proceed to step 5. Otherwise, proceed to step 9.
+
+5. Perform upgrade of the standby instance.
+
+6. Enable high-availability synchronization.
+
+7. Optionally, perform promotion of the standby instance to the latest backup synchronized from the YugabyteDB Anywhere version to which to upgrade.
```

Comment:
Here's what's in the google doc provided by dev: 

_7. Optionally, promote the standby instance with the latest backup synchronized from the YugabyteDB Anywhere version to which to upgrade._

And here's the corresponding step in the diagram:

_Promote Failover with latest Version B backup._

I'm changing step 7 to:

_7. Optionally, promote the standby instance with the latest backup synchronized from the YugabyteDB Anywhere version to which to upgrade._

Waiting for dev to review and let me know what it should be.

Reply:
@eugeneckim Could you, please, clarify if step 7 is correct as:

7. Optionally, promote the standby instance with the latest backup synchronized from the YugabyteDB Anywhere version to which to upgrade.

Reply:
> Don't quite follow - update the upgraded standby using a backup from the upgraded active?

Stated differently: This step is suggesting the user, as part of validation, can perform a test failover.  It's optional, as customers will have various exit criteria.  This specific customer wanted to test failovers as part of their upgrade validation.

> Here's what's in the google doc provided by dev:
> 
> _7. Optionally, promote the standby instance with the latest backup synchronized from the YugabyteDB Anywhere version to which to upgrade._
> 
> And here's the corresponding step in the diagram:
> 
> _Promote Failover with latest Version B backup._

"Version B" was a shortened version of "YugabyteDB Anywhere version to which to upgrade" so that the text would fit into the diagram.  It was depicted as a note since it's an example subset of "Validate promotions."

> 
> I'm changing step 7 to:
> 
> _7. Optionally, promote the standby instance with the latest backup synchronized from the YugabyteDB Anywhere version to which to upgrade._
> 
> Waiting for dev to review and let me know what it should be.

I compared the two versions of Step 7 in the comment and could not determine a difference between the two.

I went back to the original doc and the original step 7 said:

> Optionally perform promotion of Failover to with latest backup synchronized from Version B.

Comparing these two, I agree with the proposed change.

Reply:
Okay, I'm keeping the modified step 7.